content/blog/2021-12-16-extend-skywalking-with-nbb/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Extending Apache SkyWalking with non-breaking breakpoints"
+date: 2021-12-16
```

Comment:
This means this blog will be available on 12/16 only. Are you sure?

Reply:
I removed the date but I'm not sure I should have. Other blog posts have date too?

Reply:
You should keep it for the date to publish.

Reply:
Oh duh, wrong date. Fixed